scholarly/data_types.py

```diff
@@ -62,7 +62,7 @@ class PublicationSource(Enum):
     PUBLICATION_SEARCH_SNIPPET = 1
     AUTHOR_PUBLICATION_ENTRY = 2
     
-class AuthorSource(Enum):
+class AuthorSource(int, Enum):
```

Comment:
What is this int that is added?

Reply:
`AuthorSource` (and `PublicationSource`) is now a subclass of `int` in addition to `Enum`. This is the solution you had indicated in #292 - https://stackoverflow.com/a/51976841/1286923 - inheriting from `int` instead of `str` since the values are of `int` type.

Reply:
For readability of the serialized elements, perhaps we should migrate the Enums to use strings instead?

Reply:
That would be good, actually. Something like

```
AUTHOR_PROFILE_PAGE = "AUTHOR_PROFILE_PAGE"
```

Do you want me to make these changes in this PR and push them?

Reply:
Yes, please do so.

Reply:
Done!